#ifndef MODIFIERDIALOG_H
#define MODIFIERDIALOG_H

#include <QDialog>
#include "employe.h"

namespace Ui {
class ModifierDialog;
}

class ModifierDialog : public QDialog
{
    Q_OBJECT

public:
    explicit ModifierDialog(QWidget *parent = nullptr);
    ~ModifierDialog();

    void setEmploye(const Employe &employe);
    Employe getEmployeModifie() const; // âœ… Ajout de la dÃ©claration correcte
    void accept() override; // âœ… DÃ©claration de la fonction accept()

private slots:
    void on_pushButton_modifierPhoto_clicked();
    void on_pushButton_enregistrer_clicked();
    void trierEmployes(int columnIndex);  // ðŸ“Œ VÃ©rifie bien que cette ligne est prÃ©sente !


private:
    Ui::ModifierDialog *ui;
    QString cheminImage; // Pour stocker le chemin de la photo de profil
    Employe employe;
    Employe employeModifie;


};

#endif // MODIFIERDIALOG_H
